posthog/models/feature_flag/flag_status.py

```diff
@@ -85,7 +85,7 @@ def is_flag_fully_rolled_out(self, flag: FeatureFlag) -> tuple[bool, FeatureFlag
             )
         if multivariate and is_multivariate_flag_fully_rolled_out:
             return True, f'This flag will always use the variant "{fully_rolled_out_variant_name}"'
-        elif self.is_boolean_flag_fully_rolled_out(flag):
```

Comment:
Meant to push this to https://github.com/PostHog/posthog/pull/26340, caught an edge case (new test checks for this)